components/Alert.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="https://kit.fontawesome.com/272d1ec438.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="Alert.css">
+</head>
+<body>
+    <nav id="navigation">
+        <label id="navlabel">Component Library</label>
+        <ul id="navItems">
+            <a  class="navItem" href="/index.html">Home</a>
+            <a  class="navItem" href="components/Avtar.html">Documentation</a>
+            <div id="githubid"> 
+                <i class="fab fa-github"></i>
+                <a  class="github" href="https://github.com/neetukumari4858?tab=repositories">Github</a>
+            </div>
+        </ul>
+    </nav>
+    <div id="outerDiv"></div>
+        <div id="sidebar">
+            <div id="sidehead">
+                <h3>Components</h3>
+            </div>
+            <ul class="list">
+                <li class="sidebarItem"><a class="AnkerItem" href="Avtar.html">Avatar</a></li>
```

Comment:
you can give better naming like anchor-item instead of AnkerItem--> no meaning to it